sqlbrite/src/main/java/com/squareup/sqlbrite/BriteDatabase.java

```diff
@@ -283,55 +290,22 @@ public QueryObservable createQuery(@NonNull final Iterable<String> tables, @NonN
   }
 
   @CheckResult @NonNull
-  private QueryObservable createQuery(final Func1<Set<String>, Boolean> tableFilter,
-      final String sql, final String... args) {
+  private QueryObservable createQuery(Func1<Set<String>, Boolean> tableFilter, String sql,
```

Comment:
They were only `final` for use inside the anonymous `Query` class previously declared inside this method. Now they're just handed off to the `DatabaseQuery` constructor so the `final` modifier had no purpose.